Update hypershift docs + add tshooting (#2092)
timescale/docs: cloud/migrate-to-cloud/hypershift.md

problems on a busy source database.
</Highlight>

If you're migrating from an Amazon RDS database, the best option is to run

> Hypershift on an EC2 instance in the same accessibility zone as your RDS

Reviewer: ```suggestion
Hypershift on an EC2 instance in the same availability zone as your RDS
```

a few test projects of your own.
</Highlight>

It is recommended that you create an index on the `time` column of your source
database before you begin the migration. Hypershift does work without an index,
but the migration runs much slower. The simplest way to achieve this is to
create a btree index on the `time` column. However, creating an index can take
some time, as the entire table needs to be read from disk. You can create the
btree index with this command:

```sql
CREATE INDEX ON "<TABLE_NAME>" USING btree (time);
```

<highlight type="important">

> Hypershift does not support composite indexes where `time` is not the first
> indexed column.

Reviewer: Same thing with "does not support".

```suggestion
Hypershift is not able to efficiently copy and compress data when the only
index is a composite index where `time` is not the first indexed column. If you
already have such a composite index, ensure that your source database has a
plain index before you run the Hypershift migration.
```

a few test projects of your own.
</Highlight>

It is recommended that you create an index on the `time` column of your source
database before you begin the migration. Hypershift does work without an index,
but the migration runs much slower. The simplest way to achieve this is to

> create a btree index on the `time` column. However, creating an index can take

Reviewer 2: ```suggestion
create a `btree` index on the `time` column. However, creating an index can take
```

some time, as the entire table needs to be read from disk. You can create the

> btree index with this command:

Reviewer 2: ```suggestion
`btree` index with this command:
```

```sql
CREATE INDEX ON "<TABLE_NAME>" USING btree (time);
```

> <highlight type="important">

Reviewer 2: ```suggestion
<Highlight type="important">
```

Hypershift is not able to efficiently copy and compress data when the only
index is a composite index where `time` is not the first indexed column. If you
already have such a composite index, ensure that your source database has a
plain index before you run the Hypershift migration.

> </highlight>

Reviewer 2: ```suggestion
</Highlight>
```

    ```bash

>     docker run -v$(pwd)/hypershift.yml -ti timescale/hypershift:0.2 clone \

Reviewer 3: @Loquacity there should be a space between `-v` and $. 
Reference Evan's command:
```
docker run -v $(pwd)/hypershift.yml:/hypershift.yml -ti timescale/hypershift:0.2 clone \
-s "host=<> user=tsdbadmin port=33870 password=<>" \
-t "host=<> user=tsdbadmin port=33039 password=<>" \
--hypertable /hypershift.yml
```

Author: oooh! Good spot! fixed 😄

    configuration file:

    ```bash
    docker run -v $(pwd)/hypershift.yml:/hypershift.yml \

>     -ti timescale/hypershift:0.2 clone \

Reviewer: There will probably be a merge conflict here anyway, but just to be sure:

```suggestion
    -ti timescale/hypershift:0.3 clone \
```
